src/ir/function.rs

```diff
@@ -85,6 +85,7 @@ fn get_abi(cc: CXCallingConv) -> abi::Abi {
         CXCallingConv_X86FastCall => abi::Abi::Fastcall,
         CXCallingConv_AAPCS => abi::Abi::Aapcs,
         CXCallingConv_X86_64Win64 => abi::Abi::Win64,
+        CXCallingConv_Invalid => abi::Abi::C, // TODO:
```

Comment:
Instead of this, could we make this function return `Option<abi::Abi>`, commenting that it should only return `None` for objective C stuff, and asserting that in the caller?